Project.toml

```
@@ -24,6 +26,7 @@ AdvancedMHStructArraysExt = "StructArrays"
 AbstractMCMC = "4"
 DiffResults = "1"
 Distributions = "0.20, 0.21, 0.22, 0.23, 0.24, 0.25"
+FillArrays = "1.3.0"
```

I'd suggest
```suggestion
FillArrays = "1"
```
since we only need `Zeros` and that hasn't changed in these versions. Also, Turing is not compatible with FillArrays > 1.0.0.